src/meta/processors/admin/RestoreProcessor.h

```diff
@@ -35,6 +35,10 @@ class RestoreProcessor : public BaseProcessor<cpp2::ExecResp> {
   nebula::cpp2::ErrorCode replaceHostInZone(const HostAddr& ipv4From,
                                             const HostAddr& ipv4To,
                                             bool direct = false);
+
```

Comment:
by the way, not important, why haven't you formatted the comment of replaceHostInPartition ?

Reply:
I'll format it.